core/src/main/scala/Elo.scala

```diff
@@ -36,6 +35,14 @@ object Elo extends OpaqueInt[Elo]:
   def computeRatingDiff(player: Player, games: Seq[Game]): Int =
     computeNewRating(player, games) - player.rating
 
+  def getExpectedScore(ratingDiff: Int): Double =
```

Comment:
We should add some tests for this function as well.

Reply:
I added tests for `getExpectedScore`, but also i had to migrate table to BigDecimal (which is slower than Double) just for test purpose. Please consider resigning for `getExpectedScore`, we test this indirectly through `computeRatingDiff`

Reply:
We should be able to stick to Double or even Float there. I'm not sure why it wouldn't work in tests. You may want to use `assertCloseTo` like there:
https://github.com/lichess-org/scalachess/blob/master/test-kit/src/test/scala/DecayingStatsTest.scala#L26